homeassistant/components/risco/__init__.py

```diff
@@ -127,10 +127,10 @@ async def _async_setup_cloud_entry(hass: HomeAssistant, entry: ConfigEntry) -> b
     try:
         await risco.login(async_get_clientsession(hass))
     except CannotConnectError as error:
-        raise ConfigEntryNotReady() from error
-    except UnauthorizedError:
+        raise ConfigEntryNotReady from error
+    except UnauthorizedError as error:
         _LOGGER.exception("Failed to login to Risco cloud")
```

Comment:
We don't want to log an error when raising `ConfigEntryAuthFailed`. Debug level would be ok.